### PR TITLE
Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by moosichu**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+[`bug`](https://github.com/hackersatcambridge/hac-website/issues?q=is%3Aopen+is%3Aissue+label%3Abug)
+or a possible
+[`enhancement`](https://github.com/hackersatcambridge/hac-website/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement),
+and ensure that if it is a bug, that as much information as possible is there to
+help us reproduce it.
+
+# Pull Requests
+
+If the pull request should resolve any issues, please list them in a comment!
+
```
Comment (jaredkhan) [inline CONTRIBUTING.md]: Maybe this should go in the Pull Request template

Comment (jaredkhan) [inline CONTRIBUTING.md]: We should also mention that frontend changes should have screenshots

Comment (moosichu) [reply CONTRIBUTING.md]: Could you do that pls? I'm on phone now.

Comment (moosichu) [reply CONTRIBUTING.md]: Done

Comment (moosichu) [reply CONTRIBUTING.md]: Done :)

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## What the enhancement is
+
+...
+
+## Why it could be helpful
+
+...
+
+PLEASE DON'T FORGET TO APPROPRIATELY TAG THE ISSUE AS A BUG OR ENHANCEMENT!
```
Comment (Pinpickle) [inline .github/ISSUE_TEMPLATE.md]: @moosichu are you talking about labels here? AFAIK, labels can only be added by those with write access

Comment (moosichu) [reply .github/ISSUE_TEMPLATE.md]: OK, I will remove references to that then

Comment (moosichu) [reply .github/ISSUE_TEMPLATE.md]: Done

**Pull request merged**